lms/djangoapps/courseware/views/views.py

```diff
@@ -743,16 +744,23 @@ def _progress(request, course_key, student_id):
     # checking certificate generation configuration
     enrollment_mode, is_active = CourseEnrollment.enrollment_mode_for_user(student, course_key)
 
+    try:
+        user_national_id = NationalId.objects.get(user=student.id)
+        national_id = user_national_id.get_national_id()
+    except NationalId.DoesNotExist:
+        national_id = False
+
     context = {
         'course': course,
         'courseware_summary': courseware_summary,
         'studio_url': studio_url,
         'grade_summary': grade_summary,
         'staff_access': staff_access,
         'student': student,
+        'national_id': national_id,
         'passed': is_course_passed(course, grade_summary),
         'credit_course_requirements': _credit_course_requirements(course_key, student),
-        'certificate_data': _get_cert_data(student, course, course_key, is_active, enrollment_mode)
+        'certificate_data': _get_cert_data(student, course, course_key, is_active, enrollment_mode, national_id)
```

Comment:
@pepeportela Do not break API function call. You can get national_id inside the `_get_cert_data` function with the `student` variable (User).